PKHeX.Core/Resources/text/es/lang_es.txt

```diff
@@ -42,7 +42,7 @@ SAV_SimpleTrainer=Editor del Entrenador
 SAV_SuperTrain=Récords de superentrenamiento
 SAV_Trainer=Editor del Entrenador
 SAV_Trainer7=Editor de Datos del Entrenador
-SAV_Underground=Editor de Puntuación de subterráneo
+SAV_Underground=Editor de puntuación de Subterráneo
```

Comment:
Oops, my bad, it's actually Subsuelo in the games (so it would be "del Subsuelo").